app/uk/gov/hmrc/fileupload/ApplicationLoader.scala

```diff
@@ -86,6 +87,43 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
   override lazy val mode = context.environment.mode
   override lazy val runModeConfiguration = configuration
 
+  val acceptedMaxItems: Int = runModeConfiguration.getInt("constraints.accepted.maxItems")
+                                .getOrElse(throwRuntimeException("accepted.maxNumFiles"))
+  val acceptedMaxSize: String = runModeConfiguration.getString("constraints.accepted.maxSize")
+                                  .getOrElse(throwRuntimeException("accepted.maxMaxSize"))
+  val acceptedMaxSizePerItem: String = runModeConfiguration.getString("constraints.accepted.maxSizePerItem")
+                                        .getOrElse(throwRuntimeException("accepted.maxSizePerItem"))
+  val acceptedContentTypes: List[ContentTypes] = getContentTypesInList(runModeConfiguration.getString("constraints.accepted.contentTypes"))
+                                                  .getOrElse(throwRuntimeException("accepted.contentTypes"))
+
+  val defaultMaxItems: Int = runModeConfiguration.getInt("constraints.default.maxItems")
+                              .getOrElse(throwRuntimeException("default.maxNumFiles"))
+  val defaultMaxSize: String = runModeConfiguration.getString("constraints.default.maxSize")
+                                  .getOrElse(throwRuntimeException("default.maxMaxSize"))
+  val defaultMaxSizePerItem: String = runModeConfiguration.getString("constraints.default.maxSizePerItem")
+                                        .getOrElse(throwRuntimeException("default.maxSizePerItem"))
+  val defaultContentTypes: List[ContentTypes] = getContentTypesInList(runModeConfiguration.getString("constraints.default.contentTypes"))
+                                                  .getOrElse(throwRuntimeException("default.contentTypes"))
+
+  def throwRuntimeException(key: String) = {
+    throw new RuntimeException(s"default value $key need to define")
+  }
+
+  def getContentTypesInList(contentTypesInString: Option[String]): Option[List[ContentTypes]] = {
+    contentTypesInString.map(types ⇒ types.split(",").toList)
+  }
+
+  val envelopeConstraintsConfigure = EnvelopeConstraintsConfigure(acceptedMaxItems = acceptedMaxItems,
```

Comment:
You can hide most of the code to the helper object

Reply:
will be done, thanks again